api/cli_test.go

```diff
@@ -135,6 +135,175 @@ func TestCliAssignRole(t *testing.T) {
 	}
 }
 
+func TestCliJoinChannel(t *testing.T) {
+	if disableCliTests {
+		return
+	}
+
+	th := Setup().InitBasic()
+	channel := th.CreateChannel(th.BasicClient, th.BasicTeam)
+
+	// These test cannot run since this feature requires an enteprise license
```

Comment:
@crspeller would love to but the enterprise unit tests just set `utils.IsLicensed = true` which won't work because the CLI unit tests just shell how with `exec.Command("bash", ...`